lib/chdb_node.cpp

```diff
@@ -121,6 +98,34 @@ Napi::String QueryWrapper(const Napi::CallbackInfo &info) {
   return Napi::String::New(env, result);
 }
 
+// QuerySession function will save the session to the path
+char *QuerySession(const char *query, const char *format, const char *path,
```

Comment:
The old impl of queryStable and the new one can not co-exist in one process.
As the new one need to keep some global stuff, but the old one tend to cleanup everything on finished.
We should use the Connection base queryConn here instead. 
And the Query should also use the the Connection base queryConn
See also: https://github.com/chdb-io/chdb/blob/d26d2ce84190e9f9e2dbbde024ac09e9739198eb/chdb/__init__.py#L75